test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2662,18 +2715,19 @@ def forward(self, input):
         x = torch.randn(3, 4, 5)
         self.run_test(UnbindModel2(), x)
 
-    @disableScriptTest()
     def test_split(self):
         class SplitModel(torch.nn.Module):
             def forward(self, input):
-                return input.split([2, 1, 2])
+                out1, out2, out3 = input.split([2, 1, 2])
```

Comment:
I suppose this is the workaround to ensure model outputs are tensors, instead of a single tensor[] output?

Reply:
Yes, exactly.